src/assets/stylesheets/chat/typingIndicator.scss

```diff
@@ -1,4 +1,4 @@
-.container {
+.typing-indicator-container {
```

Comment:
Changed class names in few components as we had the same className for parent and its child component which resulted in unpredictable styling behaviour.